New Cppcheck errors: '[unreadVariable]'

**Checklist**
* [x] I reviewed the [Contributing Guide](https://github.com/nasa/cFS/blob/main/CONTRIBUTING.md).
* [x] I performed a cursory search to see if the bug report is relevant, not redundant, nor in conflict with other tickets.

**Describe the bug**
Latest version of Cppcheck is issuing the following failures for MD:
```
fsw/src/md_cmds.c:46:40: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32              Status          = CFE_SUCCESS;
                                       ^
fsw/src/md_cmds.c:49:40: style: Variable 'TableIndex' is assigned a value that is never used. [unreadVariable]
    uint16             TableIndex      = 0;
                                       ^
fsw/src/md_cmds.c:134:40: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32              Status          = CFE_SUCCESS;
                                       ^
fsw/src/md_cmds.c:138:40: style: Variable 'TableIndex' is assigned a value that is never used. [unreadVariable]
    uint16             TableIndex      = 0;
                                       ^
fsw/src/md_cmds.c:195:44: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32                   Status         = CFE_SUCCESS;
                                           ^
fsw/src/md_cmds.c:199:44: style: Variable 'DwellEntryPtr' is assigned a value that is never used. [unreadVariable]
    MD_DwellControlEntry_t *DwellEntryPtr  = 0; /* points to local task data */
                                           ^
fsw/src/md_cmds.c:200:44: style: Variable 'EntryIndex' is assigned a value that is never used. [unreadVariable]
    uint16                  EntryIndex     = 0;
                                           ^
fsw/src/md_cmds.c:410:40: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32                 Status       = CFE_SUCCESS;
                                       ^
fsw/src/md_dwell_pkt.c:40:37: style: Variable 'Result' is assigned a value that is never used. [unreadVariable]
    int32                    Result = 0;
                                    ^
fsw/src/md_dwell_pkt.c:42:48: style: Variable 'EntryIndex' is assigned a value that is never used. [unreadVariable]
    uint16                   EntryIndex        = 0;
                                               ^
fsw/src/md_dwell_pkt.c:43:48: style: Variable 'NumDwellAddresses' is assigned a value that is never used. [unreadVariable]
    uint16                   NumDwellAddresses = 0;
                                               ^
fsw/src/md_dwell_tbl.c:174:27: style: Variable 'Status' is assigned a value that is never used. [unreadVariable]
    int32  Status         = CFE_SUCCESS;
                          ^
fsw/src/md_utils.c:192:21: style: Variable 'OS_Status' is assigned a value that is never used. [unreadVariable]
    int32 OS_Status = OS_SUCCESS;
                    ^
##[error]Process completed with exit code 255.
```

**To Reproduce**
Run the current version of Cppcheck on the current main branch MD source code.

**Expected behavior**
Cppcheck should pass without raising any errors.

**Reporter Info**
Avi @thnkslprpt

--- a/fsw/src/md_cmds.c
+++ b/fsw/src/md_cmds.c
@@ -42,11 +42,11 @@ extern MD_AppData_t MD_AppData;
 
 void MD_ProcessStartCmd(const CFE_SB_Buffer_t *BufPtr)
 {
-    int32              ErrorCount      = 0;
-    int32              Status          = CFE_SUCCESS;
-    int32              NumTblInMask    = 0; /* Purely as info for event message */
-    uint16             TableId         = 0;
-    uint16             TableIndex      = 0;
+    int32              ErrorCount = 0;
+    int32              Status;
+    int32              NumTblInMask = 0; /* Purely as info for event message */
+    uint16             TableId      = 0;
+    uint16             TableIndex;
     MD_CmdStartStop_t *Start           = (MD_CmdStartStop_t *)BufPtr;
     bool               AnyTablesInMask = false;
 
@@ -130,12 +130,12 @@ void MD_ProcessStartCmd(const CFE_SB_Buffer_t *BufPtr)
 
 void MD_ProcessStopCmd(const CFE_SB_Buffer_t *BufPtr)
 {
-    int32              ErrorCount      = 0;
-    int32              Status          = CFE_SUCCESS;
-    int32              NumTblInMask    = 0; /* Purely as info for event message */
-    MD_CmdStartStop_t *Stop            = (MD_CmdStartStop_t *)BufPtr;
-    uint16             TableId         = 0;
-    uint16             TableIndex      = 0;
+    int32              ErrorCount = 0;
+    int32              Status;
+    int32              NumTblInMask = 0; /* Purely as info for event message */
+    MD_CmdStartStop_t *Stop         = (MD_CmdStartStop_t *)BufPtr;
+    uint16             TableId      = 0;
+    uint16             TableIndex;
     bool               AnyTablesInMask = false;
 
     for (TableId = 1; TableId <= MD_NUM_DWELL_TABLES; TableId++)
@@ -192,13 +192,13 @@ void MD_ProcessStopCmd(const CFE_SB_Buffer_t *BufPtr)
 void MD_ProcessJamCmd(const CFE_SB_Buffer_t *BufPtr)
 {
     /* Local variables */
-    int32                   Status         = CFE_SUCCESS;
+    int32                   Status;
     MD_CmdJam_t *           Jam            = 0;
     bool                    AllInputsValid = true;
     cpuaddr                 ResolvedAddr   = 0;
-    MD_DwellControlEntry_t *DwellEntryPtr  = 0; /* points to local task data */
-    uint16                  EntryIndex     = 0;
-    uint8                   TableIndex     = 0;
+    MD_DwellControlEntry_t *DwellEntryPtr; /* points to local task data */
+    uint16                  EntryIndex;
+    uint8                   TableIndex = 0;
     MD_SymAddr_t            NewDwellAddress;
 
     /*
@@ -407,7 +407,7 @@ void MD_ProcessJamCmd(const CFE_SB_Buffer_t *BufPtr)
 
 void MD_ProcessSignatureCmd(const CFE_SB_Buffer_t *BufPtr)
 {
-    int32                 Status       = CFE_SUCCESS;
+    int32                 Status;
     MD_CmdSetSignature_t *SignatureCmd = (MD_CmdSetSignature_t *)BufPtr;
     uint16                TblId        = 0;
     uint16                StringLength;

--- a/fsw/src/md_dwell_pkt.c
+++ b/fsw/src/md_dwell_pkt.c
@@ -37,11 +37,11 @@ extern MD_AppData_t MD_AppData;
 
 void MD_DwellLoop(void)
 {
-    int32                    Result = 0;
+    int32                    Result;
     uint16                   TblIndex;
-    uint16                   EntryIndex        = 0;
-    uint16                   NumDwellAddresses = 0;
-    MD_DwellPacketControl_t *TblPtr            = NULL;
+    uint16                   EntryIndex;
+    uint16                   NumDwellAddresses;
+    MD_DwellPacketControl_t *TblPtr = NULL;
 
     /* Check each dwell table */
     for (TblIndex = 0; TblIndex < MD_NUM_DWELL_TABLES; TblIndex++)

--- a/fsw/src/md_dwell_tbl.c
+++ b/fsw/src/md_dwell_tbl.c
@@ -171,7 +171,7 @@ int32 MD_ReadDwellTable(const MD_DwellTableLoad_t *TblPtr, uint16 *ActiveAddrCou
 /******************************************************************************/
 int32 MD_CheckTableEntries(MD_DwellTableLoad_t *TblPtr, uint16 *ErrorEntryArg)
 {
-    int32  Status         = CFE_SUCCESS;
+    int32  Status;
     int32  FirstErrorCode = CFE_SUCCESS;
     uint16 EntryIndex;
     int32  FirstBadIndex = -1;

--- a/fsw/src/md_utils.c
+++ b/fsw/src/md_utils.c
@@ -189,7 +189,7 @@ bool MD_Verify16Aligned(cpuaddr Address, uint32 Size)
 bool MD_ResolveSymAddr(MD_SymAddr_t *SymAddr, cpuaddr *ResolvedAddr)
 {
     bool  Valid;
-    int32 OS_Status = OS_SUCCESS;
+    int32 OS_Status;
 
     /*
     ** NUL terminate the very end of the symbol name string array as a

